src/Stratis.Bitcoin/Base/ChainState.cs

```diff
@@ -11,6 +11,10 @@ public interface IChainState
         /// <summary>ChainBehaviors sharing this state will not broadcast headers which are above <see cref="ConsensusTip"/>.</summary>
         ChainedHeader ConsensusTip { get; set; }
 
+        /// <summary>The highest stored block in the repository.</summary>
+        /// <remarks>Null if BlockStore is not enabled.</remarks>
```

Comment:
could null be avoided rather than meaning blockstore not enabled?

Reply:
Not sure it's possible so it's more readable or well designed. So having null for "not initialized" is probably the best option here.